src/errors/EntityDetailsNotFoundError.vue

```diff
@@ -0,0 +1,40 @@
+<template>
+  <Head>
+    <Title>{{ APP_TITLE_SHORT }} | {{ errorData.entityName }} not found</Title>
+  </Head>
+
+  <page-header>
+    Not Found
```

Comment:
Could be even like this?

```suggestion
    {{ errorData.entityName }} Not Found
```

Reply:
I recently did it this way and in one of the above comments Michele told me it's against the designs. I suppose we should leave Not Found here

Reply:
ok, than. Thanks for info